project/ws/app/src/lib/routes/profile-v2/routes/profile-view/profile-view.component.ts

```diff
@@ -44,7 +44,7 @@ export const MY_FORMATS = {
   },
 }
 const EMAIL_PATTERN = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/
-const MOBILE_PATTERN = /^\d{10}$/
+const MOBILE_PATTERN = /^[0]?[6789]\d{9}$/
```

Comment:
This validation to be apply in MDO as well?

Reply:
Yes! Same applied too